web/pdf_attachment_viewer.js

```diff
@@ -89,6 +89,34 @@ var PDFAttachmentViewer = (function PDFAttachmentViewerClosure() {
     /**
      * @private
      */
+    _bindPdfLink:
+        function PDFAttachmentViewer_bindPdfLink(button, content, filename) {
+      var blobUrl;
+      button.onclick = function() {
+        if (!blobUrl) {
+          blobUrl = pdfjsLib.createObjectURL(content, 'application/pdf') +
```

Comment:
Nit: Similar to e.g. https://github.com/mozilla/pdf.js/blob/a719b71e5957ee513ebf43a4ff703926c629b553/web/download_manager.js#L83-L84, you may want to add `pdfjsLib.PDFJS.disableCreateObjectURL` as the third parameter here!?